test/spec/modules/liveIntentAnalyticsAdapter_spec.js

```diff
@@ -285,34 +73,52 @@ describe('LiveIntent Analytics Adapter ', () => {
     sandbox.stub(utils, 'generateUUID').returns(instanceId);
     sandbox.stub(refererDetection, 'getRefererInfo').returns({page: url});
-    sandbox.stub(auctionManager.index, 'getAuction').withArgs({auctionId}).returns({ getWinningBids: () => winningBids });
-    events.emit(EVENTS.AUCTION_END, args);
-    clock.tick(2000);
+    sandbox.stub(auctionManager.index, 'getAuction').withArgs({auctionId: AUCTION_INIT_EVENT.auctionId}).returns({ getBidRequests: () => AUCTION_INIT_EVENT.bidderRequests });
+
+    events.emit(EVENTS.AUCTION_INIT, AUCTION_INIT_EVENT);
     expect(server.requests.length).to.equal(1);
+    expect(server.requests[0].url).to.equal('https://wba.liadm.com/analytic-events/auction-init?id=77abbc81-c1f1-41cd-8f25-f7149244c800&aid=87b4a93d-19ae-432a-96f0-8c2d4cc1c539&u=https%3A%2F%2Fwww.test.com&ts=1739969798557&pid=a123&iid=pbjs&liip=y')
 
-    let requestBody = JSON.parse(server.requests[0].requestBody);
-    expect(requestBody).to.deep.equal(expectedEvent);
+    events.emit(EVENTS.BID_WON, BID_WON_EVENT);
+    expect(server.requests.length).to.equal(2);
+    expect(server.requests[1].url).to.equal('https://wba.liadm.com/analytic-events/bid-won?id=77abbc81-c1f1-41cd-8f25-f7149244c800&aid=87b4a93d-19ae-432a-96f0-8c2d4cc1c539&u=https%3A%2F%2Fwww.test.com&auc=test-div2&auid=afc6bc6a-3082-4940-b37f-d22e1b026e48&cpm=1.5&c=USD&b=appnexus&bc=appnexus&pid=a123&iid=pbjs&ts=1739971147744&rts=1739971147806&liip=y');
+  });
+
+  it('not send any events on auction init if disabled in settings', () => {
+    liAnalytics.enableAnalytics(configWithNoAuctionInit);
+    sandbox.stub(utils, 'generateUUID').returns(instanceId);
+    sandbox.stub(refererDetection, 'getRefererInfo').returns({page: url});
+    sandbox.stub(auctionManager.index, 'getAuction').withArgs({auctionId: AUCTION_INIT_EVENT.auctionId}).returns({ getBidRequests: () => AUCTION_INIT_EVENT.bidderRequests });
+
+    events.emit(EVENTS.AUCTION_INIT, AUCTION_INIT_EVENT);
+    expect(server.requests.length).to.equal(0);
   });
 
-  it('track is called', () => {
-    sandbox.stub(liAnalytics, 'track');
+  it('not send fields that are undefined', () => {
     liAnalytics.enableAnalytics(configWithSamplingAll);
-    expectEvents().to.beTrackedBy(liAnalytics.track);
-  })
+    sandbox.stub(utils, 'generateUUID').returns(instanceId);
+    sandbox.stub(refererDetection, 'getRefererInfo').returns({page: url});
+    sandbox.stub(auctionManager.index, 'getAuction').withArgs({auctionId: AUCTION_INIT_EVENT.auctionId}).returns({ getBidRequests: () => AUCTION_INIT_EVENT.bidderRequests });
+
+    events.emit(EVENTS.AUCTION_INIT, AUCTION_INIT_EVENT);
+    events.emit(EVENTS.BID_WON, BID_WON_EVENT_UNDEFINED);
+
+    expect(server.requests.length).to.equal(2);
+    expect(server.requests[1].url).to.equal('https://wba.liadm.com/analytic-events/bid-won?id=77abbc81-c1f1-41cd-8f25-f7149244c800&aid=87b4a93d-19ae-432a-96f0-8c2d4cc1c539&u=https%3A%2F%2Fwww.test.com&pid=a123&iid=pbjs&liip=y');
```

Comment:
Comparison is in [line 84](https://github.com/LiveIntent/Prebid.js/pull/51/files#diff-8459d2a8b164c26b9ccd86f5018e31136249474edd5ff73f7d27c84d65281388R84)